app/pages/settings/email.jsx

```diff
@@ -316,13 +307,26 @@ class EmailSettingsPage extends React.Component {
             <Translate content="emailSettings.project.section" />
           </strong>
         </p>
-        <table>
+        <AutoSave resource={this.props.user}>
+          <label>
+            <input
+              type="checkbox"
+              name="project_email_communication"
+              checked={this.props.user.project_email_communication}
+              onChange={handleInputChange.bind(this.props.user)}
+            />{' '}
+            <Translate content="emailSettings.general.classify" />
+            <Translate component="p" content="emailSettings.general.note" />
```

Comment:
suggestion, not necessary - maybe move `general.note` and `general.manual` out of the `<AutoSave>` component so clicking them doesn't toggle the checkbox?